packages/token-metadata/src/tokens/tokens/tokens.ts

```diff
@@ -562,6 +562,20 @@ export default {
     },
   },
 
+  STINJ: {
+    name: 'Stride Staked Injective',
+    symbol: 'STINJ',
+    decimals: 18,
+    logo: 'stinj.png',
+    coinGeckoId: 'stride-staked-injective',
+
+    cw20: {
+      decimals: 8,
```

Comment:
Are you sure that the CW20 version has 8 decimals?

Reply:
Honestly? Not sure. I copied it from the other cw20 files. I didn't know where to find it.

Reply:
What're the decimals that the CW20 contract has? I think it should be 18, the same as the native INJ.

Reply:
Where can I find this? I tried looking for it at: https://www.mintscan.io/injective/account/inj10fd06xl4q6jp9qlhemvm6ymmm83ppj2g8rzquw